profiles/package.mask

```diff
@@ -35,11 +35,6 @@
 # Removal in 30 days.  Bug #665174.
 sys-auth/fprint_demo
 
-# Michał Górny <mgorny@gentoo.org> (23 Mar 2019)
-# Unmaintained.  Fails to build.  Probably needs a fresh snapshot.
-# Removal in 30 days.  Bug #669592.
-net-misc/casync
-
```

Comment:
Please make the package.mask change a separate commit.

Reply:
Done, thanks for the friendly reminder.